update_database_scripts/unpackDatabaseCommandLine.R

```diff
@@ -579,10 +579,23 @@ if (is_ebi_reachable()) {
       
       # validate rsids by removing faulty characters (letters, asterisks, or crosses etc. after the ID)
       studyData$variant_id <- gsub("(rs\\d*).*","\\1",as.character(studyData$variant_id))
+      
+      # if the ogValueTypes is beta and the beta unit is "NA", sets the beta unit to "unit"
+      studyData <- mutate(studyData, beta_unit = case_when(ogValueTypes != "beta" ~ beta_unit,
+                                                          ogValueTypes == "beta" & beta_unit != "NA" & !is.na(beta_unit) ~ beta_unit,
+                                                          ogValueTypes == "beta" & beta_unit == "NA" | is.na(beta_unit) ~ "unit"))
+      
+      # filter out snps that have multiple beta units for a unique combinations of trait, pValueAnnotation, betaAnnotation, ogValueType, and studyID (many of these beta units should be the same, but aren't due to typos)
+      studyData <- studyData %>%
+        group_by(trait, pvalue_description, beta_description, ogValueTypes, studyID) %>%
```

Comment:
Here is the table of all the ones that should be filtered out. https://github.com/EBISPOT/gwas-user-requests/issues/49
It looks like it might be filtering out the whole study as opposed to just the unique combination.